spanner/tests/unit/test_keyset.py

```diff
@@ -15,6 +15,7 @@
 
 import unittest
 
+import six
```

Comment:
There should be two empty lines between top-level definitions (i.e. this import and the class the follows)